AutoDarkModeApp/Pages/PageSwitchModes.xaml

```diff
@@ -109,11 +109,90 @@
                             </ComboBox>
                         </DockPanel>
 
-
                     </ui:SimpleStackPanel>
                 </ui:SimpleStackPanel>
             </uc:Card>
 
+            <uc:Card Padding="30,15,15,15" Margin="0,2,0,0">
+                <ui:SimpleStackPanel>
+                    <CheckBox
+                        Click="CheckBoxProcessBlockList_Click"
+                        Content="{x:Static p:Resources.SwitchModesCheckBoxProcessBlockList}"
+                        Margin="0,0,0,0"
+                        Name="CheckBoxBlockList"/>
+
+                    <ui:SimpleStackPanel Name="SimpleStackPanelProcessBlockList">
+                        <DockPanel Margin="0,10,0,0">
+                            <Expander
+                                DockPanel.Dock="Bottom"
+                                FlowDirection="RightToLeft"
+                                Header="{x:Static p:Resources.SwitchModesExpanderHeaderBlockedProcesses}"
+                                HorizontalAlignment="Stretch"
+                                Margin="0,10,0,10">
+                                <Expander.HeaderTemplate>
+                                    <DataTemplate>
+                                        <!--Unsure why the negative left margin is necessary here, 
```

Comment:
This is because the WPF header was never designed for somethign like this.
I actually had to completely overwrite the header template on the accent color page to get this to work, so I don't think there's an easy way to do this. Hence I think it's fine for now.